### PR TITLE
wasabiwallet: init at 1.1.5

### PR DESCRIPTION
**Pull request by mmahut**
###### Motivation for this change

Init at 1.1.5

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [X] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [X] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [X] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Assured whether relevant documentation is up to date
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Comment by mmahut** [comment]
I have renamed the package to ``wasabiwallet`` as I think it makes more sense than ``walletwasabi``.

**Comment by tilpner** [comment]
@GrahamcOfBorg build wasabiwallet

File: pkgs/applications/altcoins/wasabiwallet/default.nix [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+		for i in $(find . -type f -name '*.so') wassabee
+			do
+				patchelf --set-rpath ${stdenv.lib.makeLibraryPath [ openssl_1_1 stdenv.cc.cc.lib xorg.libX11 curl fontconfig.lib krb5 zlib dotnet-sdk ]} $i
+			done
+		patchelf --set-interpreter "$(cat $NIX_CC/nix-support/dynamic-linker)" wassabee
+		ln -s $out/opt/${pname}/wassabee $out/bin/${pname}
+		cp -v $desktopItem/share/applications/* $out/share/applications
+	'';
+
+  meta = with stdenv.lib; {
```
Comment (adisbladis) [inline pkgs/applications/altcoins/wasabiwallet/default.nix]: Missing a `platforms` attribute (since you're only supporting linux).

**Review by adisbladis — CHANGES_REQUESTED** [review]

File: pkgs/applications/altcoins/wasabiwallet/default.nix (hunk 1 above)
Comment (mmahut) [reply pkgs/applications/altcoins/wasabiwallet/default.nix]: Fixed, added linux.

**Comment by adisbladis** [comment]
@GrahamcOfBorg build wasabiwallet

**Comment by jb55** [comment]
thanks for packaging this! the commit description looks wrong: `wasabiwasabi`

**Comment by mmahut** [comment]
@jb55 good catch, thank you, brainfart during renaming :)

**Review by adisbladis — APPROVED** [review]

**Pull request merged**